### PR TITLE
Add open prop in Menu

### PR DESCRIPTION
**Pull request by oorestisime**
#### What does this PR do?

Adds support for open prop in Menu

#### Where should the reviewer start?

Menu.js

#### How should this be manually tested?
manual on storybook
#### What are the relevant issues?
#2926 
#### Do the grommet docs need to be updated?
done
#### Should this PR be mentioned in the release notes?
yes
#### Is this change backwards compatible or is it a breaking change?
compatible

### CODE REVIEW

File: src/js/components/Select/README.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ Required. Options can be either a string or an object. If an object is used, use
 
 **open**
 
-Initial state of the select component
+Control the state of the component.
```
Comment (ShimiSun) [inline src/js/components/Select/README.md]: what was wrong with the previous description? I feel like `Control the state of the component.` doesn't cover the description better, unless I'm missing something.

File: src/js/components/Menu/Menu.js [hunk 2]
```diff
@@ -32,10 +32,14 @@ class Menu extends Component {
...
   };
 
-  state = { activeItemIndex: -1, open: false };
-
   buttonRefs = {};
 
+  constructor(props) {
+    super(props);
+
+    this.state = { activeItemIndex: -1, open: props.open || false };
```
Comment (ShimiSun) [inline src/js/components/Menu/Menu.js]: should we add on the description defaults to `false`?

Comment (oorestisime) [reply src/js/components/Menu/Menu.js]: yeap!

File: src/js/components/Select/README.md (hunk 1 above)
Comment (oorestisime) [reply src/js/components/Select/README.md]: From the discussion [here](https://github.com/grommet/grommet/issues/2926#issuecomment-473341079) i thought that it is best to change the description. Not sure how to describe though. Open for suggestions.

Comment (ShimiSun) [reply src/js/components/Select/README.md]: me too. let me think about it a little.

Comment (ShimiSun) [reply src/js/components/Select/README.md]: Something like "Whether the state of the component should be open" ?

**Comment by oorestisime** [comment]
@ShimiSun  changed the wording and added default value. should be good to go

**Pull request merged**